changelog.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Product updates'
+description: 'What Mintlify recently launched'
+---
+
+<Tip>
+  To upgrade your CLI, run the command below.
+</Tip>
+```
+npm i -g mintlify@latest
+```
```

Comment:
why is this here?

Reply:
> why is this here?

first, i wanted to give users quick access to what they want/need. on second thought, we could remove this part as we mention it when relevant.